.github/workflows/pull_request.yml

```diff
@@ -4,14 +4,19 @@ on:
   pull_request:
     types: [opened, reopened, synchronize]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   tests:
     name: Test
     uses: swiftlang/github-workflows/.github/workflows/swift_package_test.yml@main
     with:
-      linux_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
-      linux_pre_build_command: apt-get update && apt-get install -y libsqlite3-dev libncurses5-dev
-      windows_exclude_swift_versions: '[{"swift_version": "5.9"}, {"swift_version": "5.10"}]'
+      linux_os_versions: '["noble", "jammy", "focal", "rhel-ubi9"]'
+      linux_pre_build_command: command -v apt >/dev/null 2>&1 && apt update && apt install -y libsqlite3-dev libncurses5-dev || (command -v yum >/dev/null 2>&1 && yum update -y && yum install -y sqlite-devel)
```

Comment:
On RedHat it's `libncurses-dev`; I'd add that too.

Reply:
```
No match for argument: libncurses-dev
Error: Unable to find a match: libncurses-dev
```

Going to revert back to not including it.

Reply:
Oh, sorry, checked again and it's `ncurses-devel`.

I was looking at the Debian metapackage before -- and on Debian we should use `libncurses-dev` rather than `libncurses5-dev`; the latter is a transitional package for `libncurses-dev`.